subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoTaskExtension.java

```diff
@@ -300,64 +336,95 @@ public FileCollection getAgentClasspath() {
      * Gets all properties in the format expected of the agent JVM argument.
      *
      * @return state of extension in a JVM argument
+     * @deprecated Use {@link #asJvmArgs()} instead.
      */
+    @Deprecated
     @Internal
     public String getAsJvmArg() {
+        return getJavaAgentArg();
+    }
+
+    /**
+     * Gets all agent properties as JVM arguments.
```

Comment:
This is more than a simple getter, can you expand upon the 2 cases a little here?